internal/pkg/kernel/kernel.go

```diff
@@ -72,28 +72,34 @@ func ListKernels() ([]string, error) {
 }
 
 func Build(kernelVersion string, root string) (string, error) {
+
 	kernelImage := path.Join(root, "/boot/vmlinuz-"+kernelVersion)
 	kernelDrivers := path.Join(root, "/lib/modules/"+kernelVersion)
 	kernelDestination := KernelImage(kernelVersion)
 	driversDestination := KmodsImage(kernelVersion)
 
 	// Create the destination paths just in case it doesn't exist
-	err := os.MkdirAll(path.Dir(kernelDestination), 0755)
-	if err != nil {
-		return "", fmt.Errorf("failed to create kernal dest: %s", err)
-	}
-
-	err = os.MkdirAll(path.Dir(driversDestination), 0755)
-	if err != nil {
-		return "", fmt.Errorf("failed to create driver dest: %s", err)
-	}
+  if  err := os.MkdirAll(path.Dir(kernelDestination), 0755) ; err != nil {
+    return "", err
```

Comment:
I would suggest to keep the error wrapping. This will be helpful for the end-user, and debugging.

```suggestion
		return "", errors.Wrap(err, "failed to create kernal dest")
```

_`errors.Wrap()` is just like `fmt.Errorf()`, but `errors.Wrap()` also includes a stack trace._